python/lsst/ts/ATWhiteLightSource/chillerModel.py

```diff
@@ -195,40 +299,65 @@ def __init__(self, log):
         self.warnings = []
 
     def __str__(self):
-        output = "Control Status: " + str(self.controlStatus) \
-            + ", Pump Status: " + str(self.pumpStatus) \
-            + ", Chiller Status: " + str(self.chillerStatus) \
-            + ", Set Temp " + str(self.setTemp) \
-            + ", Supply Temp " + str(self.supplyTemp) \
-            + ", Return Temp: " + str(self.returnTemp) \
-            + ", Ambient Temp: " + str(self.ambientTemp) \
-            + ", Process Flow: " + str(self.processFlow) \
-            + ", TEC Bank 1: " + str(self.tecBank1) \
-            + ", TEC Bank 2: " + str(self.tecBank2) \
-            + ", TE Drive Level: " + str(self.teDrivePct) \
-            + ", TE Drive Mode: " + str(self.teDriveMode) \
-            + ", Fan1 Speed: " + str(self.fan1speed) \
-            + ", Fan2 Speed: " + str(self.fan2speed) \
-            + ", Fan3 Speed: " + str(self.fan3speed) \
-            + ", Fan4 Speed: " + str(self.fan4speed) \
-            + ", Uptime: " + str(self.chillerUptime)
+        output = (
+            "Control Status: "
+            + str(self.controlStatus)
+            + ", Pump Status: "
+            + str(self.pumpStatus)
+            + ", Chiller Status: "
+            + str(self.chillerStatus)
+            + ", Set Temp "
+            + str(self.setTemp)
+            + ", Supply Temp "
+            + str(self.supplyTemp)
+            + ", Return Temp: "
+            + str(self.returnTemp)
+            + ", Ambient Temp: "
+            + str(self.ambientTemp)
+            + ", Process Flow: "
+            + str(self.processFlow)
+            + ", TEC Bank 1: "
+            + str(self.tecBank1)
+            + ", TEC Bank 2: "
+            + str(self.tecBank2)
+            + ", TE Drive Level: "
+            + str(self.teDrivePct)
+            + ", TE Drive Mode: "
+            + str(self.teDriveMode)
+            + ", Fan1 Speed: "
+            + str(self.fan1speed)
+            + ", Fan2 Speed: "
+            + str(self.fan2speed)
+            + ", Fan3 Speed: "
+            + str(self.fan3speed)
+            + ", Fan4 Speed: "
+            + str(self.fan4speed)
+            + ", Uptime: "
+            + str(self.chillerUptime)
```

Comment:
Please consider the following to improve readability (it is also probably more efficient, though that is not a important here):
```
output_strs = (
    f"Control Status: {self.controlStatus}",
    f"Pump Status: {self.pumpStatus}",
   ....
    f"Uptime: {self.chillerUptime}",
)
return ", ".join(output_strs)
```